src/main/java/com/findmyrecycling/fmrenterprise/dto/RecyclableMaterial.java

```diff
@@ -16,11 +19,17 @@ public RecyclableMaterial(Long materialId, String materialName) {
     }
 
     public RecyclableMaterial() {
-
-       this.materialId = materialId;
-       this.materialName = materialName;
+        //Removed unneeded self assignment
+      // this.materialId = materialId;
+      // this.materialName = materialName;
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.